pkg/config/config.go

```diff
@@ -22,10 +22,11 @@ type ServerConfig struct {
 }
 
 type ServerSecurityOptions struct {
-	Secure  bool                 `json:"secure"`
-	Ssl     SslOptions           `json:"ssl"`
-	UseAuth bool                 `json:"useAuth"`
-	Oauth   config2.OAuthOptions `json:"oauth"`
+	Secure      bool                 `json:"secure"`
+	Ssl         SslOptions           `json:"ssl"`
+	UseAuth     bool                 `json:"useAuth"`
+	Oauth       config2.OAuthOptions `json:"oauth"`
+	AuditAccess bool                 `json:"auditAccess"`
```

Comment:
will we ever want this off?  this pr doesn't write to the database right?  so long as it's just a log line, can it just always be on if auth is on?